core/async_job/queue.ex

```diff
@@ -54,20 +54,30 @@ defmodule AntikytheraCore.AsyncJob.Queue do
   @impl true
   defun new() :: t do
     set = :gb_sets.empty()
-    %__MODULE__{jobs: %{}, index_waiting: set, index_runnable: set, index_running: set, brokers_waiting: [], brokers_to_notify: [], abandoned_jobs: []}
+    %__MODULE__{jobs:              %{},
```

Comment:
Please write in [the same way](https://github.com/access-company/antikythera/blob/master/lib/web/conn.ex#L199) other codes are written.
* trailing comma
* position of braces
```
%__MODULE__{
  jobs:              %{},
  index_waiting:     set,
  index_runnable:    set,
  index_running:     set,
  brokers_waiting:   [],
  brokers_to_notify: [],
  abandoned_jobs:    [],
}
```

Reply:
I fixed it in https://github.com/access-company/antikythera/commit/43cce81c61461e12e362aa199679425fc58dcaa3